src/ecc/codes/surface.jl

```diff
@@ -0,0 +1,60 @@
+"""The planar surface code refers to the toric code [kitaev2003fault](@cite) in a 2D lattice with open boundaries.
```

Comment:
```suggestion
"""The planar surface code refers to the code [kitaev2003fault](@cite) in a 2D lattice with open boundaries.
```
Minor phrasing: It is not "toric" if it has open boundaries.